sandbox/install-client-archives.sh

```diff
@@ -29,8 +29,16 @@ for PACKAGE in "${CLIENT_PACKAGES[@]}"; do
     wget ${BASE_URL}${VERSION}/${PACKAGE}-${VERSION}-all.tar.gz \
         || wget ${BASE_URL}v${VERSION}/${PACKAGE}-${VERSION}-all.tar.gz
     tar xvzf ${PACKAGE}-${VERSION}-all.tar.gz
-    pip install -r ${PACKAGE}-${VERSION}-all/requirements.txt
-    pip install -r ${PACKAGE}-${VERSION}-all/requirements_gui.txt
+    REQUIREMENTS=${PACKAGE}-${VERSION}-all/requirements.txt
+    REQUIREMENTS_GUI=${PACKAGE}-${VERSION}-all/requirements_gui.txt
+
+    # requirements vary across the Python packages
+    if [ -f ${REQUIREMENTS} ]; then
```

Comment:
I think we might want to drop this `if` block -- `requirements.txt` should always exist both in pre- and post-split code, so I would probably expect this to fail if that file doesn't exist for some reason.

Reply:
Thanks for the quick review, and done.